sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -368,7 +368,7 @@ public virtual async Task<ServiceBusReceivedMessage> PeekAsync(CancellationToken
         /// </remarks>
         ///
         /// <returns>A task to be resolved on when the operation has completed.</returns>
-        public virtual async Task CompleteAsync(
+        internal virtual async Task CompleteAsync(
```

Comment:
Just curious, why did we change this to internal?

Reply:
Ramya mentioned we should leave the batch settlement out of Preview 1.